consensus/consensus_v2.go

```diff
@@ -167,6 +172,10 @@ func (consensus *Consensus) onAnnounce(msg *msg_pb.Message) {
 	consensus.pbftLog.AddMessage(recvMsg)
 	consensus.pbftLog.AddBlock(&blockObj)
 
+	if consensus.mode.Mode() == ViewChanging {
```

Comment:
add some log info and explain why returning.

Reply:
because we still want to add the message and block to catch up later if viewchanging stucked. I will add some comments